lighthouse-cli/test/smokehouse/test-definitions/tricky-metrics/expectations.js

```diff
@@ -26,6 +26,22 @@ module.exports = [
       },
     },
   },
+  {
+    lhr: {
+      requestedUrl: 'http://localhost:10200/tricky-tti-late-fcp.html',
+      finalUrl: 'http://localhost:10200/tricky-tti-late-fcp.html',
+      audits: {
+        'first-cpu-idle': {
+          // FCP at ~5 seconds out
+          numericValue: '>4900',
+        },
+        'interactive': {
+          // FCP at ~5 seconds out
+          numericValue: '>4900',
```

Comment:
same for these bounds...doesn't really differentiate between ~5 seconds and ~30 seconds, but will flakiness prevent us from putting a useful upper bound?